concepts/0008-message-id-and-threading/README.md

```diff
@@ -61,7 +61,8 @@ Message threading will be implemented as a [decorator](../0011-decorators/README
         "thid": "98fd8d72-80f6-4419-abc2-c65ea39d0f38",
         "pthid": "1e513ad4-48c9-444e-9e7e-5b8b45c5e325",
         "sender_order": 3,
-        "received_orders": {"did:sov:abcxyz":1}
+        "received_orders": {"did:sov:abcxyz":1},
+        "goal_code": "aries.credx.issue"
```

Comment:
This should be "aries.vc.issue", to be consistent with what's declared elsewhere.

Reply:
Fixed.